src/cobalt_model/frontmatter.rs

```diff
@@ -370,16 +370,11 @@ impl FrontmatterBuilder {
             self.format = Some(format);
         }
 
-        if self.published_date.is_none() || self.slug.is_none() {
+        if self.slug.is_none() {
             let file_stem = file_stem(relpath);
-            let (file_date, file_stem) = parse_file_stem(file_stem);
-            if self.published_date.is_none() {
-                self.published_date = file_date;
-            }
-            if self.slug.is_none() {
-                let slug = slug::slugify(file_stem);
-                self.slug = Some(slug);
-            }
+            let (_, file_stem) = parse_file_stem(file_stem);
```

Comment:
>  Add or update date prefix on file name
>    Set or update published_date in frontmatter
> BREAKING CHANGE: We will not longer extract dates from pages by default

these three things made a funny effect: `parse_file_stem`'s first result (`Option<DateTime>`) is never used, shall I change the function?

Reply:
I think I'm missing we the date extraction was removed?  Because we assume the date will always be there because of `cobalt publish`?  What about people doing it manually?

Reply:
Doing what manually?

`parse_file_stem`'s job it to parse the date out from the filename, but as with this PR, `publish` take total control of it (the breaking change) we don't need to keep the date if any is present (we update the date anyway)

Reply:
Not everyone might use `publish` (manually publishing).

Reply:
Oh, you mean, manually editing `is_draft` and call `build`?

I though this:
> BREAKING CHANGE: We will not longer extract dates from pages by default

was about not doing it any more by default?

Reply:
Oh got it ^^' I'm restoring this part of the code

Reply:
Weren't we talking about making this behavior conditioned on the flag in the collection?

I'm fine with us re-thinking that.  I'm just wanting to double check the plan

Reply:
I'm kinda lost. What I understand so far is that we stop processing filename by default and the flag is only for the `publish` command. I might misunderstood badly here ^^'

Reply:
Ok, my recommendation has changed on this based on recent experience and is posted on https://github.com/cobalt-org/cobalt.rs/issues/562#issuecomment-453720805.

I think that means this issue is resolved and everything is good to go?